src/fides/ctl/core/deploy.py

```diff
@@ -65,10 +65,19 @@ def compare_semvers(version_a: List[int], version_b: List[int]) -> bool:
 
 def check_docker_version() -> bool:
     """Verify the Docker version."""
+    try:
+        run("docker info", stdout=PIPE, stderr=PIPE, check=True, shell=True)
+    except CalledProcessError:
+        raise SystemExit(
+            "Error: Could not run 'docker info'. Please ensure that Docker is running and try again."
```

Comment:
`Could not run` implies to me that an error occurred attempting to execute the command, not that the command executed and exited with an error status.

```suggestion
            "Error: 'docker info' command failed. Please ensure that Docker is running and try again."
```